src/config.rs

```diff
@@ -110,6 +110,7 @@ pub enum Step {
     Juliaup,
     Kakoune,
     Krew,
+    Helm,
```

Comment:
This should be higher up due to alphabetic ordering.

Reply:
Sure, I'll fix that.
I just assumed to group it with krew/another Kubernetes tool.

Reply:
I've resorted the entire enum alphabetically.